libnavigation-base/src/main/java/com/mapbox/navigation/base/route/NavigationRoute.kt

```diff
@@ -151,19 +155,29 @@ class NavigationRoute internal constructor(
             routerOrigin: RouterOrigin,
             routeParser: SDKRouteParser = NativeRouteParserWrapper
         ): List<NavigationRoute> {
+            logI("NavigationRoute.create called for $routeRequestUrl", TAG)
```

Comment:
This does not happen very often, right? We won't spam with these logs? As far as I know, they are not removed for release builds. Is that right?

Reply:
> This does not happen very often, right?

It happens every route request

>  We won't spam with these logs? 

I want 😄 
It happens often, that a bug accrues on customer's side only and logs are the only clue.

> As far as I know, they are not removed for release builds. Is that right?

yes